leaf-ui/js/displayAnalysis.js

```diff
@@ -31,7 +40,11 @@ function displayAnalysis(analysisResults){
     // ElementList = analysisResults.elementList;
 
     // Update history log
-    updateHistory(currentAnalysis);
+    // updateHistory(currentAnalysis);
```

Comment:
@mvarnum did we make a decision on whether we are completely scrapping the history log? I can't remember -- if we basically are we should remove this line of code to clean up